code/obj/machinery/door/airlock.dm

```diff
@@ -282,18 +285,22 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 	visible = 0
 
 /obj/machinery/door/airlock/glass/command
+		name = "command airlock"
 		icon = 'icons/obj/doors/Doorcom-glass.dmi'
 		req_access = list(access_heads)
 
 /obj/machinery/door/airlock/glass/engineering
+		name = "command airlock"
```

Comment:
🤔

Reply:
I saw that an hour ago I haven't gotten to a computer!!!